compiler/src/iree/compiler/Codegen/SPIRV/Passes.cpp

```diff
@@ -403,6 +403,7 @@ void addSPIRVCooperativeMatrixVectorizePassPipeline(
 
   // Tile and distribute to GPU subgroups.
   funcPassManager.addPass(createSPIRVTileToCooperativeOpsPass());
+  funcPassManager.addPass(createPropagateDispatchSizeBoundsPass());
```

Comment:
This is needed because `createSPIRVTileToCooperativeOpsPass`  will make new bounds on thread.ids and we need those for `createRemoveSingleIterationLoopPass` to do the right thing.